xml/System.IO/File.xml

```diff
@@ -2214,10 +2217,13 @@ An I/O error occurred.</exception>
         <Parameter Name="fileHandle" Type="Microsoft.Win32.SafeHandles.SafeFileHandle" Index="0" FrameworkAlternate="net-7.0" />
       </Parameters>
       <Docs>
-        <param name="fileHandle">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="fileHandle">A <see cref="T:Microsoft.Win32.SafeHandles.SafeFileHandle" /> to the file or directory for which to obtain creation date and time information.</param>
+        <summary>Returns the creation date and time of the specified file or directory.</summary>
```

Comment:
Why `creation date and time` instead just `creation time`?

@danmoseley which form should we prefer?

```suggestion
        <summary>Returns the creation time of the specified file or directory.</summary>
```

Reply:
Precedence in this same file is to use `creation date and time`. For example, lines: 2278, 2344, 2401. I'll leave it as is, for consistence.